pkg/systemd/overview-cards/realmd.jsx

```diff
@@ -177,6 +177,7 @@ export class RealmdClient {
                     const proc = cockpit.spawn([helper, "ipa", "request", kerberos.RealmName, user],
                                                { superuser: "require", err: "message" });
                     proc.input(password);
+                    proc.catch(ex => console.warn("Failed to run", helper, "ipa request:", ex.toString()));
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18168-20230113-085018-b4a747a8-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)